src/parse/pdxfile/lexer.rs

```diff
@@ -462,20 +476,22 @@ impl Iterator for Lexer<'_> {
                                 return Some(Ok((start_i, Lexeme::General(token), i + 1)));
                             }
                             id.add_char(c);
-                            if warn_linebreaks {
-                                // Warn, but continue parsing the string.
-                                let msg = "quoted string not closed";
-                                warn(ErrorKey::ParseError).weak().msg(msg).loc(self.loc).push();
-                            }
                             self.consume();
-                        } else if Game::is_hoi4() && c == '\\' && !escaped {
+                        } else if c == '\\' && !escaped {
```

Comment:
Are you sure about removing the `is_hoi4`? Hoi4 has a different string processor. I'm not aware of the other games having escaping.